docs/changes/io.votable/15959.api.rst

```diff
@@ -0,0 +1,2 @@
+Only output user-requested columns in ``votable.parse``.
+Previously, unselected columns would just be masked (and unallocated).
```

Comment:
[The recommended way to read VOTables is through the unified I/O interface](https://docs.astropy.org/en/stable/io/votable/index.html#getting-started), so I'd expect the change log entry to describe how that is affected.

Reply:
rephrased